x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/services/analytics_service/delete_analytics.ts

```diff
@@ -39,3 +37,114 @@ export const deleteAnalytics = async (d: DataFrameAnalyticsListRow) => {
   }
   refreshAnalyticsList$.next(REFRESH_ANALYTICS_LIST_STATE.REFRESH);
 };
+
+export const deleteAnalyticsAndTargetIndex = async (
+  d: DataFrameAnalyticsListRow,
+  deleteTargetIndex: boolean,
+  deleteTargetPattern: boolean
+) => {
+  const toastNotifications = getToastNotifications();
+  const destinationIndex = d.config.dest.index;
+  try {
+    if (isDataFrameAnalyticsFailed(d.stats.state)) {
+      await ml.dataFrameAnalytics.stopDataFrameAnalytics(d.config.id, true);
+    }
+    const status = await ml.dataFrameAnalytics.deleteDataFrameAnalyticsAndTargetIndex(
+      d.config.id,
+      deleteTargetIndex,
+      deleteTargetPattern
+    );
+    if (status.analyticsJobDeleted?.success) {
+      toastNotifications.addSuccess(
+        i18n.translate('xpack.ml.dataframe.analyticsList.deleteAnalyticsSuccessMessage', {
+          defaultMessage: 'Request to delete data frame analytics {analyticsId} acknowledged.',
```

Comment:
I think this reads better with the addition of `job` in the text - `Request to delete data frame analytics job {analyticsId} acknowledged.`

Reply:
Done in https://github.com/elastic/kibana/pull/66934/commits/5bd97e9c466ad1a88d32da4f432583d5ddc3aed0